src/main/resources/assets/caverns_and_chasms/lang/ru_ru.json

```
@@ -0,0 +1,810 @@
+  "item.minecraft.smithing_template.netherite_upgrade.ingredients": "Незеритовый и некромиевый слитки",
+  "potion.whenSmashed": "Когда разбивается:",
+  "container.caverns_and_chasms.atone": "Искупление",
+  "container.caverns_and_chasms.atone.durability": "Будет утрачено %s-%s%% прочности",
+  "container.caverns_and_chasms.atone.spinel.many": "Шпинель: %s",
+  "container.caverns_and_chasms.atone.spinel.one": "Шпинель: 1",
+  "container.caverns_and_chasms.splurter": "Залповый выбрасыватель",
+  "container.caverns_and_chasms.scatterer": "Залповый раздатчик",
+  "container.caverns_and_chasms.toolbox": "Ящик с инструментами",
+  "container.caverns_and_chasms.toolbox.more": "и ещё %s...",
+  "container.caverns_and_chasms.storage_duct": "Труба-хранилище",
+  "container.caverns_and_chasms.dismantle": "Демонтировать снаряжение",
+  "container.caverns_and_chasms.dismantle.missing_smithed_item": "Поместите улучшенный предмет",
+  "container.caverns_and_chasms.dismantle.missing_spinel": "Поместите шпинель",
+  "container.caverns_and_chasms.dismantle.result_error_tooltip": "Уберите все конечные предметы",
+  "item.caverns_and_chasms.tuning_fork.capture_note": "Записана нота %s",
+  "item.caverns_and_chasms.tuning_fork.change_note": "Установлена нота %s",
+  "item.caverns_and_chasms.tuning_fork.note": "Нота",
+  "tooltip.caverns_and_chasms.latitude": "Широта",
+  "tooltip.caverns_and_chasms.longitude": "Долгота",
+  "tooltip.caverns_and_chasms.altitude": "Высота",
+  "tooltip.caverns_and_chasms.day": "День",
+  "tooltip.caverns_and_chasms.weather": "Погода",
+  "tooltip.caverns_and_chasms.clear": "Ясная",
+  "tooltip.caverns_and_chasms.rainy": "Дождливая",
+  "tooltip.caverns_and_chasms.snowy": "Снежная",
+  "tooltip.caverns_and_chasms.stormy": "Грозовая",
+  "tooltip.caverns_and_chasms.null": "Неизвестная",
+
+  "item.caverns_and_chasms.music_disc_epilogue": "Пластинка",
+
+  "painting.caverns_and_chasms.embedded.title": "Воткнутый",
+  "painting.caverns_and_chasms.exsanguinated.title": "Бескровный",
+  "painting.caverns_and_chasms.isolation.title": "Изоляция",
+  "painting.caverns_and_chasms.knight.title": "Рыцарь",
+  "painting.caverns_and_chasms.noir.title": "Нуар",
+  "painting.caverns_and_chasms.prototype_701.title": "прототип_701",
+  "painting.caverns_and_chasms.squirmy.title": "Извивание",
+  "painting.caverns_and_chasms.starry_night.title": "Звёздная ночь",
+
+  "death.attack.caverns_and_chasms.spiked_rail": "%1$s был проткнут рельсами",
+  "death.attack.caverns_and_chasms.spiked_rail.player": "%1$s был проткнут рельсами, борясь с %2$s",
+  "death.attack.caverns_and_chasms.lava_lamp": "%1$s засмотрелся на лавовую лампу",
+  "death.attack.caverns_and_chasms.lava_lamp.player": "%1$s засмотрелся на лавовую лампу, борясь с %2$s",
+  "death.attack.caverns_and_chasms.kunai": "%1$s был заколот до смерти %2$s",
+  "death.attack.caverns_and_chasms.kunai.item": "%1$s был заколот до смерти %2$s используя %3$s",

```

Для согласованности с ваниллой №3:
```suggestion
  "death.attack.caverns_and_chasms.kunai.item": "%1$s был заколот до смерти %2$s с помощью %3$s",
```